scripts/gulp/dev-server.js

```diff
@@ -37,113 +109,75 @@ function DevServer(port, config) {
-            </div>
-            <div id="iframe2-container" style="margin: 10px 0 0 75px;">
-            </div>
-            <pre style="margin: 20px 75px 75px 75px;">${readmeText()}</pre>
-            <script>
-            var appHost = document.location.hostname;
-
-            window.hypothesisConfig = function () {
-              // See https://h.readthedocs.io/projects/client/en/latest/publishers/config/
-              return {
-                // enableExperimentalNewNoteButton: true,
-                // showHighlights: 'always',
-                // theme: 'clean',
-
-                // Example focused user mode
-                // focus: {
-                //   user: {
-                //     username: 'foo',
-                //     authority: 'lms',
-                //     displayName: 'Foo Bar',
-                //   }
-                // },
-                // Example services config
-                // services: [{
-                //   apiUrl: 'http://localhost:5000/api',
-                //   authority: 'partner.org',
-                //   allowLeavingGroups: false,
-                //   groups: ['a-group-id', 'another-group-id'],
-                // }],
-
-                // Open the sidebar when the page loads
-                openSidebar: true,
-              };
-            };
-
-            var embedScript = document.createElement('script');
-            embedScript.src = '${
-              config.clientUrl
-            }'.replace('{current_host}', document.location.hostname);
-            document.body.appendChild(embedScript);
-
-            var iframeIsAdded = false;
-            var addIframeBtn = document.querySelector('#add-test');
-            if(addIframeBtn){
-              addIframeBtn.addEventListener('click', function() {
-                if (!iframeIsAdded) {
-                  var iframe1 = document.querySelector('#iframe1');
-                  var iframeNew = iframe1.cloneNode();
-                  iframeNew.src = "/document/code_of_conduct";
-                  iframeNew.id = "iframe2";
-                  iframeIsAdded = true;
-                  document.querySelector('#iframe2-container').appendChild(iframeNew);
-                } else {
-                  var iframe2 = document.querySelector('#iframe2');
-                  iframe2.parentNode.removeChild(iframe2);
-                  iframeIsAdded = false;
-                }
-              });
-            }
-            </script>
-          </body>
-          </html>
-        `;
+      switch (url.pathname) {
+        case '/document/license':
+          content = preformattedContent('./LICENSE');
+          break;
+        case '/document/code_of_conduct':
+          content = preformattedContent('./CODE_OF_CONDUCT');
+          break;
+        case '/document/doyle':
+          content = contentWithEmbeddedClient(
+            './scripts/gulp/html/doyle.html',
```

Comment:
This may be a _terrible_ place to put these source files, I have no attachment...